### DISCUSSION TITLE
Questions pertaining to defining cluster identity with scRNA seq data

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#693] [General]
Hi,

 I need clarification on a few questions on Integration of ATAC with RNA seq data.

1) Is it okay to use unsupervised/agnostic Clustering for downstream analysis?

2) If in the Constrained Integration step the pre-cluster does not identify **ALL** the cell groups or clusters found in the scRNA seq object how best to modify the parameters to get it to pick up all cell type categories labeled in the scRNA data?
3) As per the example cTNK and cNonTNK do we generate similar categories for each  cell type if we have multiple cell types in the Seurat object?
4) If we have to generate cell type (e.g., cTNK) and non-cell type (e.g., cNonTNK) categories for each unique cluster then can there be overlaps in the non-cell type categories specified?

Thank you in advance!

**Comment by rcorces** [comment]
1. I'm not sure what you mean. All clustering in ArchR is unsupervised. Perhaps you mean unconstrained integration? If so, this is dependent on your analysis and your intuition of your own data. It is ok to use unconstrained integration but depending on your experimental design and biological system it can be better to use constrained integration.
2. There is no one solution to this. Clustering is tricky and we dont have recommendations beyond what is in the manual
3. I think you are over-interpreting the tutorial. The tutorial is not the only way to do analysis and you should extrapolate from the tutorial to apply ArchR to your data in the way that makes the most sense to you. We cant give recommendations on how you should do your analysis because (1) this takes too much time and (2) every analysis is different.
4. See answer to 3 above. When doing constrained integration, I dont think there should be a situation where overlaps are present.

**Comment by sylestiel** [comment]
Thanks!!
